client/public/index.html

```diff
@@ -9,6 +9,45 @@
     <link rel="icon" type="image/png" href="/favicon.png" />
 
     <link rel="stylesheet" href="/build/bundle.css" />
+    
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/crypto-js/3.1.2/rollups/hmac-sha256.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/crypto-js/3.1.2/components/enc-base64-min.js"></script>
```

Comment:
I like to keep our dependencies local, if possible, so that we don't need to be connected to the internet to run Relm. I wonder if there's a way to re-arrange this so that the <script> below will depend on packages...